xbmc/addons/LanguageResource.cpp

```diff
@@ -29,12 +29,10 @@ namespace ADDON
 {
 
 CLanguageResource::CLanguageResource(const AddonInfoPtr& addonInfo)
-  : CResource(addonInfo, AddonType::RESOURCE_LANGUAGE)
+  : CResource(addonInfo, AddonType::RESOURCE_LANGUAGE),
+    // parse <extension> attributes
+    m_locale(CLocale::FromString(Type(AddonType::RESOURCE_LANGUAGE)->GetValue("@locale").asString()))
```

Comment:
was better readable where was before

Reply:
Like I said in the other PR, reason for the change is performance, not readability. (init members with desired value in init list)